src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx

```diff
@@ -93,16 +101,16 @@ class EduBenefitsApp extends React.Component {
       <div className="row">
         {devPanel}
         <Element name="topScrollElement"/>
-        <div className="medium-4 columns show-for-medium-up">
-          <Nav
-              data={data}
-              pages={pageState}
-              chapters={chapters}
-              currentUrl={currentLocation.pathname}/>
+        <div className="medium-8 columns">
```

Comment:
It can be in the same div, but when I tried it, the NavHeader and progress bar just got squashed together:

![image](https://cloud.githubusercontent.com/assets/12970166/22655753/fc66c6c0-ec45-11e6-8ab7-99830bea76c5.png)

I'll see what I can do to separate them while they're in the same div.